src/specification/models/curratedLocality/index.json

```diff
@@ -0,0 +1,71 @@
+{
+  "additionalProperties": false,
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Name of the locality"
+    },
+    "versionId": {
+      "type": "string",
+      "description": "Id of the specific version"
+    },
+    "createdAt": {
+      "type": "string",
+      "description": "Timestamp corresponding to when created"
+    },
+    "createdBy": {
+      "type": "string",
+      "description": "Some kind of reference to the individual that created the locality"
+    },
+    "updatedBy": {
+      "type": "string",
+      "description": "Some kind of reference to the individual that updated/created the specific version"
+    },
+    "updatedAt": {
+      "type": "string",
+      "description": "Timestamp corresponding to when updated. Could be called versionCreatedAt"
```

Comment:
This part feels like an internal comment/question: `Could be called versionCreatedAt` Should we pick one to start with?